README.md

```diff
@@ -1,19 +1,186 @@
 
-1. [Dafny](http://dafny.org/), a programming language that formally verifies your implementation matches your specification
-2. [Duvet](https://github.com/awslabs/duvet), a code quality tool for measuring how well your human-readable specification is covered in code
+## Why does code have to be correct?
 
-This content is currently a work in progress - use at your own risk, mind our dust, and stay tuned!
+If code is not correct it may not work.
+In which case you risk blocked builds.
+What if it is incorrect but everything still works?
+This can be much worse.
+
+Let's keep it positive.
+Correct software improves
+
+* Operations -
+  Your software performs as you expect, more uptime.
+* Evolution -
+  You know what needs to change, faster development.
+* Security -
+  You can reason about security properties, better sleep.
+
+## How hard can it be?
+
+The problem with correctness is it is an external property.
+You can read code, but you can not know by the code alone that it is correct.
+There are also a lot of people involved.
+You may not have the same project in multiple languages,
+but you probably have multiple projects that interoperate.
+They all need to agree on shared idea of correctness.
+
+You need external information to judge if something is correct.
+A design document is exactly what we need to define correctness!
+We can all just read the doc right?
+
+![](./img/how-hard.png)
+
+## Duvet
+
+Reading the design doc is a great first step.
+But reading it is not a mechanism to get every requirement implemented.
+Duvet helps you "remember" the requirements in the doc.
+It is a code quality tool to help bound correctness.
+Duvet connects requirements from design docs to source and test code.
+
+![](./img/duvet.png)
+
+## How does Duvet work?
+
+Duvet extracts requirements from text documents.
+It will extract MUST/SHOULD (RFC2119 normative requirements) automatically.
+It was originally built to handle RFCs,
+but we are working to natively support Markdown.
+Duvet helps you annotate your project
+with these requirements as comments.
+Duvet can then report on your project
+and tell you what is complete, what is tested, what is left to do.
+
+![](./img/duvet-works.png)
+## Persona: Designer
+
+For the designer writing the document,
+Duvet is a mechanism to get requirements implemented and tested.
+You sprinkle in `MUST` or `SHOULD` into your document,
+and you can block the build until every requirement
+not only has an implementation, but also a test.
+
+![](./img/designer.png)
+
+## Persona: Developer
+
+For the developer writing the implementation
+Duvet maintains the punch list for you,
+you can focus on getting the best implementation for a given requirement.
+As you write the code you can ask:
+Dose this code satisfy this requirement?
+Does this test verify this behavior?
+If you need details,
+the document is right there with delicious cake-like background.
```

Comment:
"cake-like" doesn't resolve in this context, expand on what you mean without using that term.